### PR TITLE
Update optimized_ops.h

### PR DESCRIPTION
**Pull request by luxupu**
Use GetReciprocal to replace the 1/(1+x) logic.

### CODE REVIEW

**Comment by alextp** [comment]
I don't understand how these two implementations are equivalent

**Comment by luxupu** [comment]
Thanks for the comment. The logic is given in the link below:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/kernels/internal/reference/softmax.h#L105-L107

**Review by jianlijianli — APPROVED** [review]
Thanks!
